adapter/parser/src/main/java/org/mobilitydata/gtfsvalidator/parser/GtfsEntityParser.java

```diff
@@ -36,23 +40,37 @@
 public class GtfsEntityParser implements GtfsSpecRepository.RawEntityParser {
     private final GtfsSpecificationProto.CsvSpecProto fileSchema;
     private final RawFileInfo rawFileInfo;
+    private final FloatValidator floatValidator;
+    private final IntegerValidator integerValidator;
+    private final DateValidator dateValidator;
 
-    public GtfsEntityParser(GtfsSpecificationProto.CsvSpecProto fileSchema, RawFileInfo rawFileInfo) {
+    private static final String DATE_PATTERN = "yyyyMMdd";
+
+    public GtfsEntityParser(GtfsSpecificationProto.CsvSpecProto fileSchema, RawFileInfo rawFileInfo,
+                            FloatValidator floatValidator, IntegerValidator integerValidator,
+                            DateValidator dateValidator) {
         this.fileSchema = fileSchema;
         this.rawFileInfo = rawFileInfo;
+        this.floatValidator = floatValidator;
+        this.integerValidator = integerValidator;
+        if (!dateValidator.isStrict()) {
+            throw new IllegalArgumentException("Date validator must be strict");
```

Comment:
Add link to strictness definition
https://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/AbstractFormatValidator.html#isStrict()